addons/website_blog/i18n/vi.po

```diff
@@ -46,7 +46,7 @@ msgstr "&lt;?xml version=\"1.0\" encoding=\"utf-8\"?&gt;"
 #. module: website_blog
 #: model_terms:ir.ui.view,arch_db:website_blog.blog_post_short
 msgid "' page header."
-msgstr "'."
+msgstr "'tiêu đề trang."
```

Comment:
@huy97852 Nó đang đúng lại sửa thành sai. Cần phải xem kỹ code cũng như mở giao diện phần mềm ra để đối chiếu.
```suggestion
msgstr "'."
```
Đây là code ở xml: https://github.com/Viindoo/odoo/blob/c05e0f86688eafa75ce05e8566d8bca4f524f9f6/addons/website_blog/views/website_blog_templates.xml#L42

Và thuật ngữ được tách thành 2 phần:
1. https://github.com/Viindoo/odoo/blob/c05e0f86688eafa75ce05e8566d8bca4f524f9f6/addons/website_blog/i18n/vi.po#L887-L888
2. https://github.com/Viindoo/odoo/blob/c05e0f86688eafa75ce05e8566d8bca4f524f9f6/addons/website_blog/i18n/vi.po#L48-L49

Khi ghép chúng nó lại với nhau thì sẽ đúng, kết quả sẽ có dạng `Sửa tiêu đề trang 'blog ABCD'.` thay vì `Sửa 'blog ABCD' tiêu đề trang.`

@Viindoo/product @Viindoo/is @Viindoo/project Anh em dịch cần lưu ý case này nhé.
@leotranviindoo @royleviindoo @phamgiang2510 Các chú lưu ý với anh em chỗ này nhé. Vì anh gặp lỗi này của anh em vài lần rồi.

Reply:
Dạ vâng ạ , em sửa lại ngay

Reply:
vâng anh.